config/initializers/git_revision.rb

```diff
@@ -1,8 +1,6 @@
 module AppInfo
   if Rails.env.production?
-    #TODO: git SHA available once devops PR #271 is merged:
-    #https://github.com/department-of-veterans-affairs/devops/pull/271
-    GIT_REVISION = "unknown"
+    GIT_REVISION = ENV["APP_GIT_SHA"]
```

Comment:
My concern is with our test coverage more than anything else. If we leave the conditional logic in, can we remove the app filters for initializers and ensure that coverage is > 90% for initializers on local and on travis? I fear not.